avm/res/operational-insights/workspace/data-export/main.bicep

```diff
@@ -0,0 +1,70 @@
+metadata name = 'Log Analytics Workspace Data Exports'
+metadata description = 'This module deploys a Log Analytics Workspace Data Export.'
+metadata owner = 'Azure/module-maintainers'
+
+// ============== //
+//   Parameters   //
+// ============== //
+
+@description('Required. The data export rule name.')
+@minLength(4)
+@maxLength(63)
+param name string
+
+@description('Conditional. The name of the parent workspaces. Required if the template is used in a standalone deployment.')
+param workspaceName string
+
+@description('Optional. Destination properties.')
+param destination object = {}
+
+@description('Optional. Active when enabled.')
+param enable bool = false
+
+@description('Optional. Enable telemetry via the Customer Usage Attribution ID (GUID).')
+param enableDefaultTelemetry bool = true
+
+@description('Optional. An array of tables to export, for example: [\'Heartbeat\', \'SecurityEvent\'].')
+param tableNames array = []
+
+// =============== //
+//   Deployments   //
+// =============== //
+
+resource defaultTelemetry 'Microsoft.Resources/deployments@2021-04-01' = if (enableDefaultTelemetry) {
```

Comment:
Remove telemetry from all children

Reply:
Removed and re-tested, commit https://github.com/Azure/bicep-registry-modules/pull/612/commits/c2dfbf20b99f619db38c62595a68cc8f31cd9812